### PR TITLE
Add a microphysics scaling parameter for calibration 

### PR DESCRIPTION
**Pull request by trontrytel**
I want to try some calibration runs where some microphysics tendencies are rescaled.

### CODE REVIEW

**Comment by trontrytel** [comment]
bors try

**Comment by trontrytel** [comment]
bors try

File: integration_tests/Manifest.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # This file is machine-generated - editing it directly is not advised
 
-julia_version = "1.7.2"
+julia_version = "1.7.0"
```
Comment (charleskawczynski) [inline integration_tests/Manifest.toml:3]: `pkg> up` should be done in separate PRs, please see [this PR](https://github.com/CliMA/TurbulenceConvection.jl/pull/893#issue-1172872109) to see how 🙂

Comment (trontrytel) [reply integration_tests/Manifest.toml:3]: I'm struggling here a little. I thought that with CLIMAParameters update being a patch release I won't have to do anything for our CI. But it could not find a new parameter. 

After I did up CLIMAParameters, it felt the need to update many packages that I did not ask to be touched. And the parameter I added is still not available
